sky/spot/spot_utils.py

```diff
@@ -523,7 +523,7 @@ def dump_job_table_cache(job_table: str):
         json.dump((time.time(), job_table), f)
 
 
-def load_job_table_cache() -> Optional[Tuple[str, str]]:
+def load_job_table_cache() -> Optional[Tuple[float, str]]:
```

Comment:
Shouldn't this be int? (Assuming this is job_id)

Reply:
The returned value is a tuple of (timestamp, job_table), where the timestamp is the time when the job table is dumped and the job_table is the dumped job table in string. None if the cache file does not exist.

Added the docstr for the return value. Thanks!